src/DGtal/geometry/doc/moduleQuickHull.dox

```diff
@@ -0,0 +1,491 @@
+   - merge possible parallel facets and update "on" points
+   - reassign "above" points of \a f to these new facets and put them into the queue
+
+3. The algorithm stops when all current facets have no "above" points.
+
+4. Optionnaly extract vertices of the convex hull polytopes by
+determining the "on" points that belong to at least \a d facets.
+
+@note In opposition with the usual `qhull` implementation, this
+class uses a kernel that can be chosen in order to provide exact
+computations. This is the case for lattice points, but also rational
+points. Computation times are very similar.
+
+@note In opposition with `CGAL 3D convex hull` package, or with the
+arbitrary dimensional `CGAL dD Triangulation` package, this algorithm does
+not build a simplicial convex hull. Facets may not be triangles or
+simplices in higher dimensions. This happens frequently for lattice
+points, where coplanar and cospherical situations are common.
+
+@note This version is generally more than twice faster than `CGAL` function
+`convex_hull_3` for the usual CGAL kernels Cartesian and
+Exact_predicates_inexact_constructions_kernel.
+
+@note However this implementation is not tailored for incremental
+dynamic convex hull computations.
+
+@note Last, the method for computing the Delaunay triangulation by
+using the convex hull in a higher dimensional space is generally much
+slower than CGAL method for computing the Delaunay triangulation by
+point location (5-10 times slower in our experience).
+
+@section dgtal_quickhull_sec2 Computing convex hulls and Delaunay cell complex using QuickHull
+
+In order to get a fine control over convex hull computations, you can
+use class QuickHull directly. It is required to choose a \b kernel
+specific to your objective. For now, the available kernels are:
+
+- \ref ConvexHullIntegralKernel : it allows the computation of the
+  convex hull of a range of lattice points in arbitrary dimension.
+
+- \ref ConvexHullRationalKernel : it allows the computation of the
+  convex hull of a range of rational points in arbitrary
+  dimension. The user fixes the precision at kernel instantiation.
+
+- \ref DelaunayIntegralKernel : it allows the computation of the
+  Delaunay cell complex of a range of lattice points in arbitrary
+  dimension.
+
+- \ref DelaunayRationalKernel : it allows the computation of the
+  Delaunay cell complex of a range of rational points in arbitrary
+  dimension. The user fixes the precision at kernel instantiation.
+
+
+@subsection dgtal_quickhull_sec21 Convex hull of lattice points
+
+To compute the convex hull of lattice points, you need to include `QuickHull.h`.
+
+\snippet geometry/tools/exampleLatticeBallQuickHull3D.cpp QuickHull3D-Includes
+
+You should then define some typedefs:
+
+\snippet geometry/tools/exampleLatticeBallQuickHull3D.cpp QuickHull3D-Typedefs
+
+Then we assume that \a V contains a range of lattice 3D points (with
+either 32 bits or 64 bits precision). You may compute their convex
```

Comment:
can the Integer type be "BigInteger"?

Reply:
not in this version, I am afraid

Reply:
Ok I have updated the four kernels. Now it handle BigInteger for internal computations and also for lattice points.